README.md

```diff
@@ -1,15 +1,41 @@
 # Foreman Ansible Modules [![Build Status](https://travis-ci.org/theforeman/foreman-ansible-modules.svg?branch=master)](https://travis-ci.org/theforeman/foreman-ansible-modules)
 
-This repository contains Ansible modules for interacting with a Foreman server API and various plugin APIs such as Katello.
+This repository contains Ansible modules for interacting with the Foreman API and various plugin APIs such as Katello.
 
 ## Goals
 
 The intent of this repository is to be a place that community members can develop or contribute modules. The goals of this repository are:
 
   * centralized location for community modules
   * a single repository to clone for interacting with Foreman & plugins
-  * an intermediate landing place for modules before pushing them to Ansible community
-  * repository maintainers will be working to push the modules into Ansible proper [https://github.com/ansible/ansible/tree/devel/lib/ansible/modules/remote_management/foreman](https://github.com/ansible/ansible/tree/devel/lib/ansible/modules/remote_management/foreman)
+  * source for the official Ansible collection (TBD)
```

Comment:
Since almost nothing on this branch is going to be part of that Ansible collection, we could reference to the master branch here.

Reply:
`s/almost//` ;)